### PR TITLE
Bug 1922280: No oVirt label in upstream

### PR DESCRIPTION
**Pull request by glekner**

### CODE REVIEW

**Comment by yaacov** [comment]
/hold
@glekner hi, I don't know why tests are failing, putting on hold, maybe we see something.
p.s.
I also have a comment in the code, PTAL

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/strings/import-providers.ts [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
-export const getProviderName = (provider: VMImportProvider) => titleResolver[provider];
+export const getProviderName = (provider: VMImportProvider) => {
+  const isUpstream = window.SERVER_FLAGS.branding === 'okd';
+  return titleResolver[provider](isUpstream);
+};
 
-export const getProviderEndpointName = (provider: VMImportProvider) =>
-  endpointTitleResolver[provider];
+export const getProviderEndpointName = (provider: VMImportProvider) => {
+  const isUpstream = window.SERVER_FLAGS.branding === 'okd';
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/strings/import-providers.ts:23]:
```suggestion
  const isUpstream = window?.SERVER_FLAGS?.branding === 'okd';
```

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/strings/import-providers.ts [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
-  [VMImportProvider.VMWARE]: 'vCenter',
+  [VMImportProvider.OVIRT]: (isUpstream: boolean) =>
+    isUpstream ? 'the oVirt/RHV API' : 'the RHV API',
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  [VMImportProvider.VMWARE]: (isUpstream: boolean) => 'vCenter',
 };
 
-export const getProviderName = (provider: VMImportProvider) => titleResolver[provider];
+export const getProviderName = (provider: VMImportProvider) => {
+  const isUpstream = window.SERVER_FLAGS.branding === 'okd';
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/strings/import-providers.ts:18]:
```suggestion
  const isUpstream = window?.SERVER_FLAGS?.branding === 'okd';
```

**Comment by glekner** [comment]
fixed @yaacov

**Pull request merged**